data/strings/strings.txt

```diff
@@ -23843,20 +23897,28 @@
     ru:few = %d трека
     ru:one = %d трек
     ru:other = %d треков
-    sk = %d sledovania
+    sk:one = %d sledovanie
+    sk:other = %d sledovania
     sr:few = %d путање
     sr:one = %d путања
     sr:other = %d путања
-    sv = %d spår
-    sw = Njia %d
-    th = %d ติดตามต่าง ๆ
-    tr = %d rota
+    sv:one = %d spår
+    sv:other = %d spår
+    sw:one = Njia %d
+    sw:other = Njia %d
+    th:one = %d ติดตาม
+    th:other = %d ติดตามต่าง ๆ
+    tr:one = %d rota
+    tr:other = %d rotalar
     uk:few = %d треки
     uk:one = %d трек
     uk:other = %d треків
-    vi = %d các chặn đường
-    zh-Hans = %d 个轨迹
-    zh-Hant = %d 個軌跡
+    vi:one = %d chặng đường
+    vi:other = %d các chặng đường
```

Comment:
here its different forms (somewhere above there were identical samples)

Reply:
I didn't get the problem here.